lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_clear_intermediate_tags.rb

```diff
@@ -5,18 +5,20 @@ def self.run(params)
         UI.message("Deleting tags for version: #{params[:version]}")
 
         require_relative '../../helper/git_helper.rb'
-        Fastlane::Helper::Ios::GitHelper.delete_tags(params[:version])
 
-        # Cleanup local tags and refetch them to make sure we're up-to-date with remote
-        local_tags = Fastlane::Helper::GitHelper.list_local_tags()
```

Comment:
The root problem here is that we're calling with `delete_on_remote: true`, but we can simplify this quite a bit.

Reply:
Yeah, I agree. I haven't seen weird tags pushed to the repo for a while now. If they restart to appear, we can re-add this check.

Reply:
But, we may still want to download the remote tags, otherwise we may not delete tags which have been pushed to GitHub, but are not on the local machine.

Reply:
Good point – added in 395a1fabd5357f9485466c4cd3e5e33b9e3d3afd